core/pubnub_coreapi_ex.c

```diff
@@ -27,12 +27,13 @@
 struct pubnub_publish_options pubnub_publish_defopts(void)
 {
     struct pubnub_publish_options result;
-    result.store      = true;
-    result.cipher_key = NULL;
-    result.replicate  = true;
-    result.meta       = NULL;
-    result.method     = pubnubSendViaGET;
-    result.ttl        = SIZE_MAX;
+    result.store               = true;
+    result.cipher_key          = NULL;
+    result.replicate           = true;
+    result.meta                = NULL;
+    result.method              = pubnubSendViaGET;
+    result.ttl                 = SIZE_MAX;
```

Comment:
In pubnub_publish_defopts method description there is mention of all default values, except tll and custom_message_type. I think it could be worth to add those as well.

Reply:
Sure, we can do it :)